content/file/utils_test.go

```diff
@@ -327,12 +344,33 @@ func Test_extractTarDirectory(t *testing.T) {
 					if string(gotContent) != wantContent {
 						t.Errorf("file content = %s, want %s", gotContent, wantContent)
 					}
+
+					header := headers[path]
```

Comment:
I'm not a maintainer, but this is the kind of thing that make table tests bad. This may in fact be the way to go, but the hardlink tests make a lot more sense to me.

Reply:
+1 to Terry's point, can we move this test case to a separate test function to make it clearer?

Reply:
👍 fixed.